torch/csrc/StorageMethods.cpp

```diff
@@ -80,14 +80,42 @@ static PyObject* THPStorage_copy_(
   END_HANDLE_TH_ERRORS
 }
 
-static PyObject* THPStorage_isPinned(PyObject* self, PyObject* noargs) {
+static PyObject* THPStorage_isPinned(PyObject* self, PyObject* args) {
```

Comment:
Wait I think we can just remove this from C++ and implement it in Python. We would just do exactly what you're doing below in the non-CUDA case--wrap the storage with a tensor and call `Tensor.is_pinned`. `Tensor.is_pinned`  supports the CUDA case as well

Reply:
> Wait I think we should just remove this from C++ and implement it in Python. We would just do exactly what you're doing below in the non-CUDA case--wrap the storage with a tensor and call `Tensor.is_pinned`. `Tensor.is_pinned` supports the CUDA case as well

I have updated this PR as you suggest， and the documentation for is_pinned and pin_memory have been added as well.
@kurtamohler  would you please review it ?